apps/package.json

```diff
@@ -152,7 +152,7 @@
     "react": "15.3.2",
     "react-addons-test-utils": "15.3.0",
     "react-bootstrap": "0.30.1",
-    "react-color": "2.2.5",
+    "react-color": "^2.11.7",
```

Comment:
FWIW I'm in favor of moving to caret-style range-dependencies like this again, since `yarn.lock` now properly fixes our dependencies to consistent versions (and lenient ranges in `package.json` will let Yarn dedupe similar dependencies more effectively). When we were using npm it was impossible to ensure deterministic dependency versions throughout our infrastructure unless we pinned to non-ranged versions (or unless we used `npm shrinkwrap` which was broken). See #3146 for more history on that.